src/odemis/model/_metadata.py

```diff
@@ -211,3 +211,8 @@
 # Rotation for FastEM multi-beam and single-beam scanner
 MD_SINGLE_BEAM_ROTATION = "Single-beam rotation"
 MD_MULTI_BEAM_ROTATION = "Multi-beam rotation"
+
+# Scan gains and offset for the FastEM scanner and descanner
+MD_SCAN_OFFSET = "Scan offset"
+MD_SCAN_GAIN = "Scan gain"
+MD_SCAN_GAIN_DYNAMIC = "Dynamic scan gain"
```

Comment:
Could you add a comment after each line describing the type. Is that a single float, or a list? Is there a fixed number of entries?

Reply:
@pieleric I've updated it, can you have another look?